lib/osf-components/addon/components/contributor-list/unregistered-contributor/template.hbs

```diff
@@ -0,0 +1,79 @@
+{{assert
+      'ContributorList::UnregisteredContributor @contributor must be unregistered and @nodeId is required!'
+      (and @contributor.unregisteredContributor @nodeId)
+}}
+
+<span
+    role='button'
+    tabindex='0'
+    data-analytics-name='Open claim unregistered user dialog'
+    data-test-unregistered-contributor-name={{@contributor.unregisteredContributor}}
+    {{on 'click' this.showDialog}}
+    {{on 'keypress' this.onKeyPress}}
```

Comment:
Can we add an aria-attribute here to indicate what this interactable element does?